src/cluster_migrateslots.c

```diff
@@ -457,7 +625,13 @@ void clusterCommandSyncSlotsEstablish(client *c) {
                 addReplyErrorObject(c, shared.syntaxerr);
                 goto cleanup;
             }
-            source_node = clusterLookupNode(c->argv[4]->ptr, CLUSTER_NAMELEN);
+            source_node_name = c->argv[4]->ptr;
```

Comment:
To be order agnostic, I believe this should be...

```suggestion
            source_node_name = c->argv[i + 1]->ptr;
```

Reply:
You are right, good catch